### PR TITLE
docs: improve the README and mark indent-delta & indent-width-cont as deprecated

### LINKED ISSUE #62: [rule] indent-delta & indent-width-cont
I'm quite sure `indent-delta` rule isn't used anymore as it doesn't make sense anymore after your refactor. If that's the case, could you delete the references to it across the project (and maybe document it in the "htmllint migration" section of your README)?

I'm not so sure about `indent-width-cont`. Does it still apply?

### PR DESCRIPTION
**Pull request by acalvo**
Closes #62

### CODE REVIEW

**Comment by acalvo** [comment]
@KamiKillertO I tried to simplify the README, as I thought that it was too wordy. Feel free to reject any/all the changes if you don't like them! :)

**Comment by KamiKillertO** [comment]
Thank you, I'll take a look after my vacation

File: README.md [hunk 1]
```diff
@@ -7,182 +7,119 @@
...
 
-LintHTML is a fork of [htmllint](https://github.com/htmllint/htmllint) with an integrated CLI, all htmllint's rules can be used with LintHTML.
-The migration from htmllint to LintHTML is easy, just follow this [guide](htmllint-migration).
-<!-- _LintHTML uses [htmlparser2](https://www.npmjs.com/package/htmlparser2) to parse your html._ -->
+LintHTML is a fork of [htmllint](https://github.com/htmllint/htmllint) featuring a built-in CLI and multiple bug fixes and improvements.
+The migration from htmllint to LintHTML is easy, as all htmllint's rules can be used with LintHTML – just follow this [simple guide](#migrate-from-htmllint).
 
 ## Installation and Usage
 
-Prerequisites:
```
Comment (KamiKillertO) [inline README.md]: We should keep the prerequisites and clearly indicate the node versions supported

File: README.md [hunk 2]
```diff
@@ -7,182 +7,119 @@
...
-* [Node.js](https://nodejs.org/en/) 6+
-* [npm](https://npmjs.com) version 3+
-
-There are two ways to install LintHTML: globally and locally.
-
-### Local Installation and Usage
-
-If you want to include LintHTML as part of your project's build system, we recommend installing it locally. You can do so using npm or yarn:
+You can install LintHTML either locally or globally.
+For most cases we recommend the former, which can be achieved this way with npm:
```
Comment (KamiKillertO) [inline README.md]: Nice but we should include a link to a doc file about `installation and usage`

File: README.md (hunk 1 above)
Comment (acalvo) [reply README.md]: Do we? Most projects don't state it in the `README` as it is not vital information (as long as it works with the latests Node.js versions, of course). In case you need more specific info, people can look in the `package.json` (engines -> node). Actually, the `README` info isn't accurate: it states that works with Node 6 onwards (6+) and actually doesn't run in Node 6. If you still prefer to keep it, we should update it to 8+, shouldn't we?

File: README.md (hunk 2 above)
Comment (acalvo) [reply README.md]: You mean a independent file, don't you? Sounds OK.

I deleted the global installation part because it makes little sense these days that we're able to run things with npx, and I'm not sure if it's a good practice to encourage people to install it globally. Anyway, if you're thinking in an extended `installation and usage` doc, it can fit there :).

Comment (KamiKillertO) [reply README.md]: A independent file yes.
Most people will use `npx` yes but not everyone, that's why I think it could be a good idea to have a separate file with differents installations and complex usage examples.

File: README.md (hunk 1 above)
Comment (KamiKillertO) [reply README.md]: The current `README` is highly inspired by the ESlint one which have a prerequisites part. But I don't really mind removing it.

Comment (acalvo) [reply README.md]: In the case of ESLint it makes more sense due to a specific minor version requirement. They say that it requires  `^8.10.0`, which means that 8.0.0 doesn't work and that they are aware of it. If that was the case for LintHTML I'd support noting that information, but as far as I know we aren't aware of any 8+ version incompatibilities, so I'd just drop it.

File: README.md (hunk 2 above)
Comment (acalvo) [reply README.md]: Done! Please take a look and tell me if you'd add anything else :).

**Comment by KamiKillertO** [comment]
:-1: thanks @acalvo
